scapy/tools/automotive/isotpscanner.py

```diff
@@ -21,6 +21,25 @@
 from scapy.contrib.isotp import ISOTPScan  # noqa: E402
 
 
+def do_scan(socket, start_id, end_id, extended, noise_listen_time,
+            sniff_time, piso, interface_string, verbose):
+    print("Start scan (%s - %s)" % (hex(start_id), hex(end_id)))
```

Comment:
No this output will look exactly the same for usual scan and extended scan. Is this intended?

Reply:
yes